examples/src/main/java/com/hedera/hashgraph/sdk/examples/consensus/SimpleWorkflow.java

```diff
@@ -1,27 +1,48 @@
 package com.hedera.hashgraph.sdk.examples.consensus;
 
-import java.nio.charset.Charset;
-
 import com.hedera.hashgraph.sdk.HederaException;
+import com.hedera.hashgraph.sdk.consensus.UpdateTopicTransaction;
+import com.hedera.hashgraph.sdk.crypto.ed25519.Ed25519PrivateKey;
 import com.hedera.hashgraph.sdk.examples.ExampleHelper;
 
+import java.nio.charset.Charset;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+
 public final class SimpleWorkflow {
 
-	private SimpleWorkflow() {
+    private SimpleWorkflow() {
     }
 
     public static void main(String[] args) throws HederaException {
         var client = ExampleHelper.createHederaClient();
+        var newKey = Ed25519PrivateKey.generate();
 
-        var memo = "My topic";
-        var message = "Message #1";
-        var topicId = client.createTopic(memo);
+        var topicId = client.createTopic("test1");
         System.out.println("Created topic: " + topicId);
 
         var topicInfo = client.getTopicInfo(topicId);
         System.out.println("Retrieved topic info: " + topicInfo);
 
-        var transactionId = client.submitMessage(topicId, message.getBytes(Charset.forName("UTF-8")));
+        var transactionId = client.submitMessage(topicId, "Message #1".getBytes(Charset.forName("UTF-8")));
+        System.out.println("Submitted message: " + transactionId);
+
+        var newAccount = client.createAccount(newKey.getPublicKey(), 10);
```

Comment:
Currently getting an error here `INSUFFICIENT_TX_FEE`. Need to resolve before merge

Reply:
client.setMaxTransactionFee(10_000_000L) before createAccount() should get past this.

Reply:
This fixed it